### PR TITLE
chore: update to moc 0.7.6

### PR DESCRIPTION
**Pull request by crusso**
* updates motoko submodule and doc to 0.7.6
* update interpreter to match.
* added MAINTENANCE.md describing what to do next time round.

The base doc was missing doc for Timers.mo. Couldn't figure out how to add it so it can come with the next release (or we create branch for just that purpose).

It would be nice if we didn't have to include the interpreter source but could download it from GH/CDN. But my JS fails.me

(I had to push the moc-0.7.6 tag to motoko-base to get the interpreter to work)

### CODE REVIEW

File: docs/references/transient/Timers.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Timer
```
Comment (ggreif) [inline docs/references/transient/Timers.md:1]: But this is the `base` documentation, not the Motoko one?!

File: sidebars.js [hunk 2]
```diff
@@ -117,6 +117,7 @@ const sidebars = {
                 "developer-docs/build/cdks/motoko-dfinity/compatibility",
                 "developer-docs/build/cdks/motoko-dfinity/stablememory",
                 "developer-docs/build/cdks/motoko-dfinity/heartbeats",
+                "developer-docs/build/cdks/motoko-dfinity/timers",
```
Comment (ggreif) [inline sidebars.js:120]: aaah, this is the Motoko one.

**Review by ggreif — APPROVED** [review]
looks good, but I couldn't check the rendering, as it failed to build 🙁

**Comment by crusso** [comment]
Renders fine, interpeter works after pushing tag to base.
I had to give up on adding the based documentation for Timer.mo, but we have the other doc for now.

**Comment by crusso** [comment]
@sesi200 @ggreif PTAL

**Review by ggreif — APPROVED** [review]
I am happy!

**Review by sesi200 — APPROVED** [review]

**Pull request merged**